docs/introduction.md

```diff
@@ -109,6 +111,30 @@ docker run \
   --db-host franklin-database
 ```
 
+Additional API options and functionality can also be configured via the `franklin` CLI* by passing options to the `serve` command:
```

Comment:
When everything compiles and `decline` is happy with how options are set up, `serve --help` will print both the CLI flags and the environment variables. If you think it would be more helpful for future readers, you could copy the environment variable help here instead of `*`-ing people into a note. The env variable names you've chosen are pretty clear / map pretty easily so I don't know if it would be helpful, but I think your perspective on what someone else starting Franklin for the first time would need is likely to be more accurate than mine.

Reply:
I think it's alright to leave them off the list here, and keep the intro guide focused on getting setup in a more direct way as it is now. I imagine in the future the docs will get more expanded and there will be more room to get into all the options.